src/main/java/net/snowflake/ingest/streaming/internal/DataValidationUtil.java

```diff
@@ -35,11 +35,9 @@
 /** Utility class for parsing and validating inputs based on Snowflake types */
 class DataValidationUtil {
   static final int MAX_STRING_LENGTH = 16 * 1024 * 1024;
-  static final int MAX_BINARY_LENGTH = 8 * 1024 * 1024;
 
-  static final BigInteger MAX_BIGINTEGER = BigInteger.valueOf(10).pow(38);
-  static final BigInteger MIN_BIGINTEGER =
-      BigInteger.valueOf(-1).multiply(BigInteger.valueOf(10).pow(38));
+  private static final int BYTES_16_MB = 16 * 1024 * 1024;
```

Comment:
what's the difference between this and MAX_STRING_LENGTH?

Reply:
There is no difference, but 16MB is not just max string length, but also max variant, object and array length, so I wanted to be explicit about the size